vtkext/private/module/vtkF3DConsoleOutputWindow.cxx

```diff
@@ -1,23 +1,11 @@
 #include "vtkF3DConsoleOutputWindow.h"
 
 #include <vtkObjectFactory.h>
-#include <vtkWindows.h>
```

Comment:
without an ifdef ? wth

Reply:
It's probably checked inside this header directly